.github/workflows/_reusable.rust.yml

```diff
@@ -37,6 +37,33 @@ jobs:
     container:
       image: xaynetci/yellow:v9
     timeout-minutes: 20
+    env:
+      XAYN_INGESTION__NET__BIND_TO: 127.0.1.1:3030
+      INGESTION_URI: http://127.0.1.1:3030
+      XAYN_PERSONALIZATION__NET__BIND_TO: 127.0.1.1:3031
+      PERSONALIZATION_URI: http://127.0.1.1:3031
+    services:
+      elasticsearch:
+        image: elasticsearch:8.5.3
+        env:
+          - discovery.type=single-node
+          - xpack.security.enabled=false
+        options: >-
+          --health-cmd curl -sf 'http://localhost:9200/_cluster/health'
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+      postgres:
+        image: postgres
```

Comment:
❓ is there the possibility to also set a fixed version? if so, we can also apply it to the `web-api/compose.db.yml`.

Reply:
yes normal docker image versioning so `postgres:15` or `postgres:15.1` should work